Sample/Kitchen Sink/DemonstrationTools/UnitConverter.swift

```diff
@@ -0,0 +1,184 @@
+//
+//  Shape+Extensions.swift
+//  flight-ui-ios
+//
+//  Created by Appivate 2023
+//
+
+import Foundation
+import SwiftUI
+import FlightUI
+
+struct UnitConverter: View {
+    @EnvironmentObject var theme: Theme
+    @StateObject var themeManager = ThemeManager(current: .dark)
+    @ObservedObject var demonstrationVM = UnitConverterViewModel()
+    @State private var weightValuesSwapped = false
+
+    var body: some View {
+        ScrollView {
+
+            VStack {
+                HeadingView(
+                    title: demonstrationVM.weightTitle,
+                    subTitle: demonstrationVM.weightSubtitle)
+
+                HStack {
+                    if  weightValuesSwapped {
+                        kgInputField
+                        swapButton
+                        lbInputField
+                    } else {
+                        lbInputField
+                        swapButton
+                        kgInputField
+                    }
+                    convertButton1
+                }
+            }
+            .padding([.top], themeManager.current.padding.grid2x)
+
+            HStack {
+                VStack {
+                    HeadingView(
+                        title: demonstrationVM.adjustableConversionTitle,
+                        subTitle: demonstrationVM.adjustableConversionSubTitle)
+                    HStack {
+                        lengthInput
+                        lengthInputUnitPicker
+                    }
+
+                    HStack {
+                        calculationResult
+                        lengthOutputUnitPicker
+                    }
+                }
+                VStack {
+                    Spacer()
+                    convertButton2
+                        .padding([.bottom], themeManager.current.padding.grid2x)
+                }
+            }
+            .padding([.top], themeManager.current.padding.grid2x)
+        }
+        .padding([.leading, .trailing], themeManager.current.padding.grid2x)
```

Comment:
Super minor, should be padding 3x to align correctly with the gutters (🤔  might be nice to have an alias for that... e.g. `padding.gutter`), can also change it to `.padding([.horizontal],...)` as well, but that's less than super minor haha